docs/docs/references/applications/README.mdx

```diff
@@ -0,0 +1,67 @@
+import ApplicationTypesIntro from '../../fragments/_application_types_intro.mdx';
+
+# 📱 Applications
+
+## Introduction
+
+Each real-world application that requires user authentication using Logto needs to be registered as an application in Logto.
+
+## Application ID
+
+_Application ID_ is a unique auto-generated identifier to identify your application with the Logto application.
+
+## Application Types
+
+We provide three application types to help developers integrate their applications with Logto:
+
+<ApplicationTypesIntro />
+
+## Application Settings
+
+### Application Name
+
+_Application Name_ is a human-readable name of the application and will be displayed in the admin console.
+
+### Description
+
+A brief description of the application will be displayed on the admin console application details page.
+
+### Authorization Endpoint
+
+_Authorization Endpoint_ is an OIDC term, and it is used to interact with the Logto service and obtain an authorization grant.
```

Comment:
may be interact with Logto service is not precise enough? As the authorization endpoint it self is part of the Logto Service